projects/i18n/types/language.ts

```diff
@@ -61,7 +61,7 @@ export interface TuiLanguageKit {
     /**
      * [@string 'choose day', @param 'choose range', @param 'choose days']
      */
-    mobileCalendarTexts: [string, string, string];
+    mobileCalendarTexts: [string, string, string] | null;
```

Comment:
The same comments

https://github.com/taiga-family/taiga-ui/pull/8264#discussion_r1696704986